src/components/pages/Auth/Auth.js

```diff
@@ -4,11 +4,23 @@ import authRequests from '../../../Helpers/data/authRequests';
 import './Auth.scss';
 
```

Comment:
Delete line 2 of this file.  Getting an error PropTypes is defined but never used